platform-core-module/src/main/java/com/softicar/platform/core/module/program/ProgramStarter.java

```diff
@@ -40,12 +42,9 @@ public static void main(String[] arguments) throws Throwable {
 
 	public void start() {
 
-		try {
+		try (var scope = new CurrentUserScope(AGUser.getSystemUser())) {
 			prepareExecution();
 			program.executeProgram();
-		} catch (Exception exception) {
```

Comment:
i don't think this `catch` should not be removed

Reply:
I restored the catch block